android/src/main/java/com/horcrux/svg/VirtualView.java

```diff
@@ -240,68 +239,57 @@ void restoreCanvas(Canvas canvas, int count) {
     canvas.restoreToCount(count);
   }
 
-  @ReactProp(name = "name")
```

Comment:
All those `@ReactProp` annotations in components are already done in `RenderableViewManager.java`.